bazel/astore/defs.bzl

```diff
@@ -151,7 +151,7 @@ def astore_download_and_extract(ctx, digest, stripPrefix, path = None, uid = Non
 
     # Check digest of archive
     checksum_args = ["sha256sum", f]
-    res = ctx.execute(checksum_args, timeout = 10)
+    res = ctx.execute(checksum_args, timeout = 60)
```

Comment:
I'd go for the gusto and increase this to 600 seconds.  We'd rather that presubmits succeed slowly than fail unnecessarily, yeah?